azerothcore/azerothcore-wotlk
# feat(Core/Config): Enable/Disable Debug BG/Arena

## CHANGES PROPOSED:
-  Add the possibility to enable Debug Arena so that you can easily do tests without having to use the command
-  Add the possibility to enable Debug Battleground so that you can easily do tests without having to use the command


## ISSUES ADDRESSED:
- Ref https://github.com/azerothcore/azerothcore-wotlk/issues/2830



## TESTS PERFORMED:
- Builds
- Tested in-game



## HOW TO TEST THE CHANGES:
1. Activate config
2. Try to use command -> See it fails
3. Queue for BG/Arena



## KNOWN ISSUES AND TODO LIST:


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
